### PR TITLE
Set selected visualisation if added to global store

### PR DESCRIPTION
**Pull request by emilom**
Saves the selected visualisation when you refresh the visualisation editor page. Will also save the visualisation if you go to an visualisation in the data explorer you have already added.

### CODE REVIEW

File: src/components/atoms/VisualisationSelector.tsx [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 type Props = {
+    default: VisualisationType;
     label: string;
     options: Record<string, string>;
     onChange: React.ChangeEventHandler<HTMLSelectElement>;
 };
 
 const VisualisationSelector: React.FC<Props> = (props) => (
-    <SelectField label={props.label} defaultValue={Object.values(props.options)[0]} onChange={props.onChange}>
+    <SelectField label={props.label} value={props.default} onChange={props.onChange}>
```
Comment (FredrikAugust) [inline src/components/atoms/VisualisationSelector.tsx:13]: Why did you remove defaultValue?

Comment (emilom) [reply src/components/atoms/VisualisationSelector.tsx:13]: The default value would only work if you re-freshed the page or visited it for the first time.

**Review by FredrikAugust — APPROVED** [review]
